src/py/flwr/server/app.py

```diff
@@ -354,10 +365,29 @@ def run_superlink() -> None:
             sys.exit(f"Fleet IP address ({address_arg}) cannot be parsed.")
         host, port, is_v6 = parsed_address
         address = f"[{host}]:{port}" if is_v6 else f"{host}:{port}"
+
+        data = _try_setup_client_authentication(args, certificates)
```

Comment:
Random idea to use a slightly more meaningful name here:
```suggestion
        maybe_keys = _try_setup_client_authentication(args, certificates)
```

Reply:
@danieljanes changed data and its callers to maybe_keys.